common/utils/licenses.tsx

```diff
@@ -29,122 +32,179 @@ const additionalData = {
+          Read more about this licence
+        </a>
+        .
+      </p>
     ),
   },
   'cc-by-nd': {
     icons: ['cc', 'ccBy', 'ccNd'],
     description: 'Free to use with attribution. No modifications permitted.',
+    humanReadabaleText: (
+      <p>
+        You can use this work for any purpose, including commercial uses. You
+        should provide attribution to the original work, source and licence. If
+        you make any modifications to or derivatives of the work, it may not be
+        distributed.{' '}
+        <a href="https://creativecommons.org/licenses/by-nd/4.0/">
+          Read more about this licence
+        </a>
+        .
+      </p>
+    ),
   },
   'cc-by-sa': {
     icons: ['cc', 'ccBy'],
+    humanReadableText: (
+      <p>
+        You can use this work for any purpose, including commercial uses,
+        without restriction under copyright law. You should also provide
+        attribution to the original work, source and licence. If you distribute
+        this work, you must use the same licence as the original.{' '}
+        <a href="https://creativecommons.org/licenses/by-sa/4.0/">
+          Read more about this licence
+        </a>
+        .
+      </p>
+    ),
   },
   'cc-by-nc-sa': {
     icons: ['cc', 'ccBy', 'ccNc'],
+    humanReadableText: (
+      <p>
+        You can copy and distribute this work, as long as it is not primarily
+        intended for or directed to commercial advantage or monetary
+        compensation. You should also provide attribution to the original work,
+        source and licence. If you distribute this work, you must use the same
+        licence as the original.{' '}
+        <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">
+          Read more about this licence
+        </a>
+        .
+      </p>
+    ),
   },
   ogl: {
     icons: [],
+    humanReadableText: (
+      <p>
+        You have permission to make copies of this work under an Open Government
+        License. This licence permits unrestricted use, distribution, and
+        reproduction in any medium, provided the original author and source are
+        credited.{' '}
+        <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/">
+          Read more about this licence
+        </a>
+        .
+      </p>
+    ),
   },
   opl: {
     icons: [],
+    humanReadableText: (
+      <p>
+        You are free to copy, publish, distribute and transmit the information;
+        adapt the information; exploit the information commercially and
+        non-commercially. You should also provide attribution to the original
+        work, source and licence.{' '}
+        <a href="https://www.parliament.uk/site-information/copyright-parliament/open-parliament-licence/">
+          Read more about this licence
+        </a>
+        .
+      </p>
+    ),
   },
   inc: {
     icons: [],
+    humanReadableText: (
+      <>
```

Comment:
Looks like an extra line inserted?

Reply:
The empty tags `<>` are used called [Fragments](https://react.dev/reference/react/Fragment), they're a React component used for grouping elements together in situations where we can only return one element!